<?xml version="1.0"?>
<launch>
    <node pkg="map_exporter" type="map_exporter_node" name="map_exporter_node" output="screen" respawn="false">
        <param name="cloud_file_path" value="/home/hkw/loopmap_20251111_095156.pcd"/>
        <param name="output_path" value="/home/hkw/dataset/Xingyun-test"/>
    </node>

</launch>